pokerpoll/pokerpoll.py

```diff
@@ -62,18 +90,28 @@ def __init__(self, bot):
                 await self.bot.say(repr(reaction))
                 await self.bot.say(reaction)
                 await self.bot.say(user)
-                
 
-            good_boys = "\n".join(boy.mention for boy, status in react_hist.items() if status == True)
-            bad_boys = "\n".join(boy.mention for boy, status in react_hist.items() if status == False)
-            dash = "\n" + "-"*30 + "\n"
-            edit_string = f"@here poker time?\n\ni play to win kid{dash}{good_boys}\n\npoker's for nerds{dash}{bad_boys}"
+            good_boys = "\n".join(
+                boy.mention for boy, status in react_hist.items() if status)
+            bad_boys = "\n".join(
+                boy.mention for boy, status in react_hist.items() if not status)
+
+            edit_string = ("@here poker time?\n\n"
+                f"i play to win kid {self.dash}{good_boys}\n\n"
+                f"poker's for nerds {self.dash}{bad_boys}")
 
             await self.bot.edit_message(poll, new_content=edit_string)
-            
+
+            now = dt.now()
+            if now.hour >= playhour and now.minute >= playminute:
```

Comment:
you can compare datetime objects directly as they implement a `__lt__` and `__gt__` function, e.g. `if now > playtime`

Reply:
see https://github.com/python/cpython/blob/master/Lib/datetime.py#L729